Sources/AppcuesKit/Presentation/Actions/Appcues/AppcuesSubmitFormAction.swift

```diff
@@ -30,12 +30,11 @@ internal class AppcuesSubmitFormAction: ExperienceAction, ExperienceActionQueueT
               let stepIndex = experienceRenderer.getCurrentStepIndex(),
               let stepState = experienceData.state(for: stepIndex) else { return }
 
-        let interactionProperties = [
+        let interactionProperties = LifecycleEvent.properties(experienceData, stepIndex).merging([
```

Comment:
order flipped here, since intention is for the new interaction props to have priority in key conflict